tests/ocean/test_compute_flow.py

```diff
@@ -215,3 +237,47 @@ def test_compute_multi_inputs():
         [compute_ddo, access_ddo],
         algo_ddo=algorithm_ddo,
     )
+
+
+def test_compute_trusted_algorithms():
+    setup = Setup()
+
+    # Setup algorithm meta to run raw algorithm
+    algorithm_ddo = get_registered_algorithm_ddo(
+        setup.publisher_ocean_instance, setup.publisher_wallet
+    )
+    _ = setup.publisher_ocean_instance.assets.resolve(algorithm_ddo.did)
+
+    algorithm_ddo_v2 = get_registered_algorithm_ddo(
+        setup.publisher_ocean_instance, setup.publisher_wallet
+    )
+    _ = setup.publisher_ocean_instance.assets.resolve(algorithm_ddo_v2.did)
+
+    # Dataset with compute service
+    compute_ddo = get_registered_ddo_with_compute_service(
+        setup.publisher_ocean_instance,
+        setup.publisher_wallet,
+        trusted_algorithms=[algorithm_ddo.did],
+    )
+    # verify the ddo is available in Aquarius
+    _ = setup.publisher_ocean_instance.assets.resolve(compute_ddo.did)
+
+    # For debugging.
```

Comment:
Please uncomment this. It is important we test trusted_algorithms in both cases: the successful one, and the unsuccessful one.